vector/src/main/java/im/vector/app/features/contactsbook/ContactsBookViewEvents.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.contactsbook
+
+import im.vector.app.core.platform.VectorViewEvents
+import im.vector.app.features.discovery.IdentityServerWithTerms
+
+sealed class ContactsBookViewEvents : VectorViewEvents {
```

Comment:
worlds smallest nit: could be a sealed interface, saves having to write `()` in the extends :sweat_smile:

Reply:
I think at the beginning of the project sealed interface was not a thing, but now, yes. I prefer to perform a global replace on a dedicated PR

Reply:
(created https://github.com/vector-im/element-android/issues/4588 about it)